docs/mmd/swap-in-sequence.md

```diff
@@ -20,7 +20,6 @@ B-->>A: pay claim invoice
 Note over B: broadcast claim tx
 else refund cooperatively
 Note over B: sign coop claim tx
```

Comment:
This line seems also useless, the `coop_close` message contains the key so that the maker can construct the `claim_tx` himself.

Reply:
Yes, I agree.
Deleted. https://github.com/ElementsProject/peerswap/pull/245/commits/d2e6a312d6e786cb09496f53724f9a89cd26e0d0